src/types/User.ts

```diff
@@ -38,6 +38,10 @@ export default interface User extends CreatedUpdatedProps, AuthorizationActions
   mobileLastChangedOn: Date;
   lastSelectedCarID?: string;
   authorizationID?: string;
+  importedData?: {
+    autoActivateUserAtImport: boolean;
+    autoActivateTagAtImport: boolean;
```

Comment:
We cannot have Tags when importing users, right?

Reply:
done ✅